zone/client_mods.cpp

```diff
@@ -337,7 +340,8 @@ int32 Client::CalcMaxHP()
 		}
 	}
 
-	return max_hp;
+	this->max_hp = std::min(max_hp, 2147483647LL);
```

Comment:
Shouldn't we do something like `std::numeric_limits<decltype(Mob::max_hp)>::max()`

Reply:
We can sure.

Reply:
I changed this in fba333f